### PR TITLE
Link contributing guide to Element Web

### PR DESCRIPTION
**Pull request by weeman1337**
## Checklist

* [x] Tests written for new code (and old code if feasible)
* [x] Linter and other CI checks pass
* [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

## PR list

* https://github.com/vector-im/element-web/pull/23153
* https://github.com/matrix-org/matrix-js-sdk/pull/2621
* https://github.com/matrix-org/matrix-react-sdk/pull/9220
* https://github.com/matrix-org/eslint-plugin-matrix-org/pull/30

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,10 +46,10 @@ All code lands on the `develop` branch - `master` is only used for stable releas
 **Please file PRs against `develop`!!**
 
 Please follow the standard Matrix contributor's guide:
-https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.md
+https://github.com/vector-im/element-web/blob/develop/CONTRIBUTING.md
 
 Please follow the Matrix JS/React code style as per:
-https://github.com/matrix-org/matrix-react-sdk/blob/master/code_style.md
+https://github.com/vector-im/element-web/blob/develop/code_style.md
```
Comment (weeman1337) [inline README.md:52]: See https://github.com/vector-im/element-web/pull/23153

File: README.md [hunk 2]
```diff
@@ -46,10 +46,10 @@ All code lands on the `develop` branch - `master` is only used for stable releas
 **Please file PRs against `develop`!!**
 
 Please follow the standard Matrix contributor's guide:
```
Comment (turt2live) [inline README.md:48]:
```suggestion
We use the same contribution guide as Element. Check it out here:
```

File: README.md [hunk 3]
```diff
@@ -46,10 +46,10 @@ All code lands on the `develop` branch - `master` is only used for stable releas
...
-https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.md
+https://github.com/vector-im/element-web/blob/develop/CONTRIBUTING.md
 
 Please follow the Matrix JS/React code style as per:
```
Comment (turt2live) [inline README.md:51]:
```suggestion
Our code style is also the same as Element's:
```

**Review by turt2live — APPROVED** [review]
otherwise lgtm

**Pull request merged**